src/main/java/emu/grasscutter/server/packet/recv/HandlerClientLockGameTimeNotify.java

```diff
@@ -11,6 +12,8 @@ public final class HandlerClientLockGameTimeNotify extends PacketHandler {
     @Override
     public void handle(GameSession session, byte[] header, byte[] payload) throws Exception {
         var packet = ClientLockGameTimeNotify.parseFrom(payload);
-        session.getPlayer().getWorld().lockTime(packet.getIsLock());
+//        session.getPlayer().getWorld().lockTime(packet.getIsLock());
+        //TODO: figure out what to implement here
+        if (packet.getIsLock()) Grasscutter.getLogger().warn("Tell Nazrin to PANIK because ClientLockGameTimeNotify is TRUE!");
```

Comment:
```suggestion
        if (packet.getIsLock()) Grasscutter.getLogger().warn("Invalid 'ClientLockGameTimeNotify' received; value is true. (please report to development channel)");
```
consider changing to something more reasonable, example shown above